src/chat/chatReducers.js

```diff
@@ -154,6 +190,52 @@ export default (state: ChatState = initialState, action: Action) => {
         last_edit_timestamp: action.edit_timestamp,
       }));
 
+    case EVENT_UPDATE_MESSAGE_FLAGS:
```

Comment:
Does this have to be here if we didn't have the webview? If so, maybe a separate PR.